src/test/java/org/codehaus/mojo/versions/utils/PropertiesVersionsFileReaderTest.java

```diff
@@ -0,0 +1,36 @@
+package org.codehaus.mojo.versions.utils;
+
+import junit.framework.TestCase;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.*;
+
+public class PropertiesVersionsFileReaderTest extends TestCase {
+
+    private static final String TEST_PROPERTIES_FILE = "src/test/resources/org/codehaus/mojo/versions/utils/testPropertiesVersionsFile.properties";
+
+    @Test
+    public void testRead() {
+        PropertiesVersionsFileReader reader = new PropertiesVersionsFileReader(TEST_PROPERTIES_FILE);
+        try {
```

Comment:
I wouldn't catch the execption here. In case of an exception, the test case should fail IMHO